.github/workflows/e2e_tests.yml

```diff
@@ -121,20 +193,18 @@ jobs:
             --base master \
             --json number \
             -q '.[0].number')" >> $GITHUB_OUTPUT
-
+      
       - name: Update pull request comment
         if: (success() || failure()) && steps.pr_number.outputs.PR_NUMBER != ''
         uses: thollander/actions-comment-pull-request@v3
         with:
           file-path: ./e2e-tests/corpus/report.md
```

Comment:
Doesn't have to come in this PR, but it would be nice to combine the reports into one comment.

For now anyway, let's include the protocol in the report, so we can distinguish them.